### PR TITLE
[NEW] : backport improve navbar #73

### PR DESCRIPTION
**Pull request by Aashray446**
### DEMO
pr-3

### CODE REVIEW

**Comment by Aashray446** [comment]
@Dnouv I saw that you have assigned the issue to me, I have raised a PR for that issue :sweat_smile:

File: app/components/menubar/newMenuBar.js [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+                    </Col>
+                  </Row>
+                  {dropDown._id === nav_Item.id && dropDown.show ? (
+                    <div>
+                      <div>
+                        {nav_Item.sub_menus.data.map(
+                          (item, key) =>
+                          (
+                            <>
+                              <div key={key} className={'p-2 fw-medium' + ` ${item.attributes.style === 'disable' ? styles.disabled : ''}`}>
```
Comment (Dnouv) [inline app/components/menubar/newMenuBar.js:149]: Could we change the style for disabled and link items? 

image

In the above image, the `disabled` item (`GSoC 2022`), which serves as a heading for categories (`GSoC 2022 Demo Day`), is not easily distinguishable. 

Another note:

Please keep the Nav Links in the left corner:
image
 (Something like above)

File: app/components/menubar/newMenuBar.js [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+                width={132}
+              />
+            </Navbar.Brand>
+          </Offcanvas.Header>
+          <Offcanvas.Body>
+            {nav_Items?.map((nav_Item) =>
+              nav_Item.url ? (
+                <div key={nav_Item.id}>
+                  <Row
+                    className={`${styles.dropdown} d-flex flex-row justify-content-between align-items-center mt-3 `}
```
Comment (Dnouv) [inline app/components/menubar/newMenuBar.js:92]: This Error occurs when opening the Nav items

image

**Review by Dnouv — CHANGES_REQUESTED** [review]

**Comment by Dnouv** [comment]
Thanks for the PR. Nice work @Aashray446 :)

I have left some comments, please take a look.

**Comment by Aashray446** [comment]
> Thanks for the PR. Nice work @Aashray446 :)
> 
> I have left some comments, please take a look.

Yeah Sure, I will fix those issues and will also suggest something for the disabled nav links asap.

**Comment by Aashray446** [comment]
@Dnouv 
image
image

Any one of them is ok ? or should I experiment with color ?

**Comment by Dnouv** [comment]
The second one looks good. Just a minor addition, add a hover effect whenever the user hovers over the links ("GSoC 2022 Demo Day"); it should turn to "Blue"/"Red" color to indicate they are links.

Thank you!

**Comment by Aashray446** [comment]
@Dnouv I have made the changes and solved the error, Here is the current working Screenshot.
I have rebased and then commit the changes, Should I make a new PR? 
pr-5

**Comment by Dnouv** [comment]
Thanks for the PR & Congrats 🎉 on your first PR. LGTM!

**Pull request merged**